KonyAccelerate/actions/mobile/appEvents.js

```diff
@@ -1,6 +1,5 @@
 define({
     AS_AppEvents_c081992218f44bf89cef587f76549c96: function AS_AppEvents_c081992218f44bf89cef587f76549c96(eventobject) {
         var self = this;
-        appPreShow();
```

Comment:
is the action removed? or only deleted the line. please check and confirm..

Reply:
This action is deleted